testkit/testkit.go

```diff
@@ -234,3 +236,28 @@ func WithPruneMode(tk *TestKit, mode variable.PartitionPruneMode, f func()) {
 	tk.MustExec("set global tidb_partition_prune_mode=`" + string(mode) + "`")
 	f()
 }
+
+// MockGC is used to make GC work in the test environment.
+func MockGC(tk *TestKit) (string, string, string, func()) {
```

Comment:
It seems that this function doesn't be used. Could we remove it?

Reply:
It was called by attributes_sql_test.go for many times.
like: https://github.com/pingcap/tidb/blob/6495a5a116a016a3e077d181b8c8ad81f76ac31b/ddl/attributes_sql_test.go#L252

Reply:
It will lead to cycle import when ddl_test use ```testkit```. so I will remove it at #32058.